rectools/visuals/metrics_app.py

```diff
@@ -218,21 +217,30 @@ def _make_chart_data_avg(self) -> pd.DataFrame:
         meta_data = self.data[meta_data_columns].drop_duplicates()
         return metrics_data.merge(meta_data, on=Columns.Model, how="left").reset_index(drop=True)
 
-    def _create_chart(
+    @staticmethod
+    @lru_cache
+    def _trim_metadata(raw_string: str, splitter: str = ", ") -> tp.Tuple[str, str]:
```

Comment:
Let's call it `_split_to_meta_and_model`

Reply:
Done